tests/cases/standalone/common/types/interval/interval.result

```diff
@@ -73,204 +73,13 @@ SELECT current_time() + INTERVAL '1 hour';
-('2022-11-01 00:00:01', INTERVAL '11 year 11 days 11 hours 11 minutes 11 seconds 11 milliseconds 11 microseconds 11 nanoseconds'),
-('2022-12-01 00:00:01', INTERVAL '12 year 12 days 12 hours 12 minutes 12 seconds 12 milliseconds 12 microseconds 12 nanoseconds');
-
-Affected Rows: 13
-
-SELECT * FROM intervals;
-
-+---------------------+-------------------------------------------------------------+
-| ts                  | interval_value                                              |
-+---------------------+-------------------------------------------------------------+
-| 2022-01-01T00:00:01 | 0 years 12 mons 0 days 0 hours 0 mins 0.000000000 secs      |
-| 2022-01-01T00:00:02 | 0 years 12 mons 0 days 0 hours 0 mins 0.000000000 secs      |
-| 2022-02-01T00:00:01 | 0 years 26 mons 0 days 0 hours 0 mins 0.000000000 secs      |
-| 2022-03-01T00:00:01 | 0 years 36 mons 0 days 3 hours 0 mins 0.000000000 secs      |
-| 2022-04-01T00:00:01 | 0 years 48 mons 0 days 0 hours 4 mins 0.000000000 secs      |
-| 2022-05-01T00:00:01 | 0 years 60 mons 0 days 0 hours 0 mins 5.000000000 secs      |
-| 2022-06-01T00:00:01 | 0 years 72 mons 0 days 0 hours 0 mins 0.006000000 secs      |
-| 2022-07-01T00:00:01 | 0 years 84 mons 0 days 0 hours 0 mins 0.000007000 secs      |
-| 2022-08-01T00:00:01 | 0 years 96 mons 0 days 0 hours 0 mins 0.000000008 secs      |
-| 2022-09-01T00:00:01 | 0 years 108 mons 9 days 0 hours 0 mins 0.000000000 secs     |
-| 2022-10-01T00:00:01 | 0 years 120 mons 0 days 10 hours 10 mins 10.010010010 secs  |
-| 2022-11-01T00:00:01 | 0 years 132 mons 11 days 11 hours 11 mins 11.011011011 secs |
-| 2022-12-01T00:00:01 | 0 years 144 mons 12 days 12 hours 12 mins 12.012012012 secs |
-+---------------------+-------------------------------------------------------------+
-
-SELECT DISTINCT interval_value FROM intervals ORDER BY interval_value;
-
-+-------------------------------------------------------------+
-| interval_value                                              |
-+-------------------------------------------------------------+
-| 0 years 12 mons 0 days 0 hours 0 mins 0.000000000 secs      |
-| 0 years 26 mons 0 days 0 hours 0 mins 0.000000000 secs      |
-| 0 years 36 mons 0 days 3 hours 0 mins 0.000000000 secs      |
-| 0 years 48 mons 0 days 0 hours 4 mins 0.000000000 secs      |
-| 0 years 60 mons 0 days 0 hours 0 mins 5.000000000 secs      |
-| 0 years 72 mons 0 days 0 hours 0 mins 0.006000000 secs      |
-| 0 years 84 mons 0 days 0 hours 0 mins 0.000007000 secs      |
-| 0 years 96 mons 0 days 0 hours 0 mins 0.000000008 secs      |
-| 0 years 108 mons 9 days 0 hours 0 mins 0.000000000 secs     |
-| 0 years 120 mons 0 days 10 hours 10 mins 10.010010010 secs  |
-| 0 years 132 mons 11 days 11 hours 11 mins 11.011011011 secs |
-| 0 years 144 mons 12 days 12 hours 12 mins 12.012012012 secs |
-+-------------------------------------------------------------+
-
--- ts + interval
-SELECT ts + interval_value as new_value from intervals;
```

Comment:
We can still add some cases for interval operation. We can select a timestamp/date and add/sub an interval without creating a table.
```
select '2022-01-01T00:00:01'::timestamp + '1 days'::interval;
```
We can test intervals like day/hour/minitue......